src/babel-external.js

```diff
@@ -0,0 +1,124 @@
+import hash from 'string-hash'
+
+import transform from '../lib/style-transform'
+import {MARKUP_ATTRIBUTE_EXTERNAL} from './_constants'
+import {
+  getExpressionText,
+  restoreExpressions,
+  makeStyledJsxCss,
+  isValidCss,
+  makeSourceMapGenerator,
+  addSourceMaps
+} from './_utils'
+
+const visitor = ({
+  path,
+  styleId,
+  types: t,
+  validate = false,
+  state
+}) => {
+  if (!path.isTemplateLiteral() || path.isStringLiteral()) {
+    return
+  }
+  const css = getExpressionText(path)
+  const prefix = `[${MARKUP_ATTRIBUTE_EXTERNAL}~="${styleId}"]`
+  const isTemplateLiteral = Boolean(css.modified)
+  const useSourceMaps = Boolean(state.file.opts.sourceMaps)
+
+  let globalCss = css.modified || css
+
+  if (validate && !isValidCss(globalCss)) {
```

Comment:
if we want to avoid this we could require people to tag external stylesheets like so:

```js
export default styledJsx`
  div { color: red }
`
```